src/util-device.c

```diff
@@ -271,6 +271,36 @@ const char *LiveGetShortName(const char *dev)
     return live_dev->dev_short;
 }
 
+int CheckAFPacketIPSDevs(void)
```

Comment:
why is this in util-device?

Reply:
I thought that since we're looping for ifaces and that kinda logic seemed to already be here in this file so this might be the right place.